"""
This is not part of the original code but adding custom exception_with_test will increase debug capability
"""


class ProfessorError(Exception):
    pass


class EnrollmentError(Exception):
    pass


class CourseError(Exception):
    pass


class StudentError(Exception):
    pass


class AddressError(Exception):
    pass
